cypress/integration/home_page_spec.js

```diff
@@ -1,15 +1,38 @@
+// to start the test run npx cypress open into your terminal
 describe('Home page user flow', () => {
     beforeEach(() => {
         cy.onLoad()
     });
-// this first test is skipet because we don't have all the JSX elements yet
-    it.skip('Should contain the Naviagtion bar with all elements ', () => {
-        cy.get()
+// 
+    it('Should contain a Header (the Navigation bar) with a How to and a Image ', () => {
+        cy.get('.header')
           .should('be.visible')
-          .get()
-          .contains()
+          .get('.header-image')
+          .should('be.visible')
+          .get('.how-to')
+          .contains('How To')
+
+        //   we need to add test for the animations
+    });
+
+    it('Should be able to click on How to and go to the How To page', () => {
+        cy.get('a')
+          .click()
+          .get('.all-pokemon')
+          .should('not.exist')
+          .get('.help-page')
+          .should('be.visible')
+    });
 
-        //   we need to add all the elements that will contain in the home page - later also test for the animations
+    it('Should be able to click on the x button and get back to the home page', () => {
+        cy.get('a')
+          .click()
+          .get('.x-icon')
+          .click()
+          .get('.help-page')
+          .should('not.exist')
+          .get('.all-pokemon')
+          .should('be.visible')
     });
```

Comment:
Same thing here witht he url, but I love that you added this step. It makes our test very robust